de.metas.procurement.base/src/main/java/de/metas/procurement/base/order/async/PMM_GenerateOrders.java

```diff
@@ -66,7 +64,20 @@ public Result processWorkPackage(final I_C_Queue_WorkPackage workPackage, final
 	private List<I_PMM_PurchaseCandidate> retrieveItems()
 	{
 		final I_C_Queue_WorkPackage workpackage = getC_Queue_WorkPackage();
-		return Services.get(IQueueDAO.class).retrieveItemsSkipMissing(workpackage, I_PMM_PurchaseCandidate.class, ITrx.TRXNAME_ThreadInherited);
+		return Services.get(IQueueDAO.class).retrieveAllItems(workpackage, I_PMM_PurchaseCandidate.class);
+	}
+
+	private List<I_PMM_PurchaseCandidate> getItemsWithQty(List<I_PMM_PurchaseCandidate> candidates)
+	{
+		List<I_PMM_PurchaseCandidate> filteredList = new ArrayList<I_PMM_PurchaseCandidate>();
+		for (I_PMM_PurchaseCandidate candidate : candidates)
+		{
+			if (candidate.getQtyOrdered().compareTo(BigDecimal.ZERO) > 0)
+			{
+				filteredList.add(candidate);
+			}
```

Comment:
* explicitly log the IDs of the `PMM_PurchaseCandidate` which you skip here
  * use Loggables.withLogger(logger, Level.DEBUG) for that, to make sure it's in the async-workpackage-log as well as in the actually logback-log